libsql-client/src/lib.rs

```diff
@@ -188,8 +188,17 @@ impl Connection {
     ) -> Self {
         let username = username.into();
         let pass = pass.into();
+        let url = url.into();
+        // Auto-update the URL to start with https://
+        let url = if url.starts_with("https://") {
+            url
+        } else if let Some(stripped) = url.strip_prefix("http://") {
+            "https://".to_owned() + stripped
+        } else {
+            "https://".to_owned() + &url
```

Comment:
I think we should add `https://` only when the url does not already contain `://` it might create a solid confusion when someone uses `file://something` url and then gets an error that `https://file://` is wrong. We should probably print error that url has to start with https:// if user gives us url that contains `://` and it does not start with `https://` or `http://`.